builder/amazon/common/access_config.go

```diff
@@ -47,23 +44,9 @@ func (c *AccessConfig) Config() (*aws.Config, error) {
 	}, nil
 }
 
-// Region returns the aws.Region object for access to AWS services, requesting
-// the region from the instance metadata if possible.
 func (c *AccessConfig) Region() (string, error) {
-	if c.RawRegion != "" {
```

Comment:
I don't believe this code should have been removed. Region() is used by non-chroot builders which do not have access to the metadata.